packages/cli/src/lib/implementation/config-middleware.ts

```diff
@@ -1,13 +1,16 @@
 import { readCodePushupConfig } from '@code-pushup/core';
 import { GlobalOptions, globalOptionsSchema } from '@code-pushup/models';
-import { ArgsCliObj, CommandBase } from './model';
+import { CommandBase, TerminalArgsObj } from './model';
 
-export async function configMiddleware<T extends ArgsCliObj>(processArgs: T) {
+export async function configMiddleware<T extends TerminalArgsObj>(
+  processArgs: T,
+) {
   const args = processArgs as T;
   const { config, ...cliOptions }: GlobalOptions =
     globalOptionsSchema.parse(args);
   const importedRc = await readCodePushupConfig(config);
   const cliConfigArgs = readCoreConfigFromCliArgs(processArgs);
+  console.log('importedRc: ', importedRc);
```

Comment:
Leftover from debugging, I assume? We can log it of course, but only when `--verbose` is set, which is out of scope for this PR, I suppose.

```suggestion
```